<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>janela_editar_treino</class>
 <widget class="QMainWindow" name="janela_editar_treino">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	background-color: #4A90E2;
}

QLabel{
	font-family: &quot;Segoe UI&quot;;
	font-size: 16px;
    color: #fff;
}

#_txt_treino{
	font-size: 26px;
	font-weight: bold;
}

#_txt_usuario{
	font-size: 26px;
	font-weight: bold;
}

#_txt_terca{
	font-size: 16px;
	font-weight: bold;
}

#_txt_quarta{
	font-size: 16px;
	font-weight: bold;
}

#_txt_segunda{
	font-size: 16px;
	font-weight: bold;
}

#_txt_quinta{
	font-size: 16px;
	font-weight: bold;
}

#_txt_sexta{
	font-size: 16px;
	font-weight: bold;
}

#_txt_comentario{
	font-size: 16px;
	font-weight: bold;
}

QPushButton{
    background: #4A90E2;
    border: 2px solid #fff;
    border-radius: 10px;
    padding: 10px 20px;

	font-family: &quot;Segoe UI&quot;;
	font-weight: bold;
	font-size: 18px;
    color: #fff;
}

QPushButton:hover{
    background: #53A3FF;
    border: 2px solid #fff;
    border-radius: 10px;
    padding: 10px 20px;
}

QTextEdit {
    background: #4A90E2; /* Cor de fundo semelhante à paleta */
    border: 1px solid #fff; /* Borda branca */
    border-radius: 8px; /* Bordas arredondadas */
    padding: 10px; /* Espaçamento interno */

    font-family: &quot;Segoe UI&quot;; /* Fonte semelhante aos outros elementos */
    font-size: 16px; /* Tamanho da fonte */
    color: #fff; /* Cor do texto branca */
}

QTextEdit:focus {
    background: #53A3FF; /* Cor de fundo ao ganhar foco */
    border: 1px solid #fff; /* Mantém a borda branca */
    border-radius: 8px; /* Mantém bordas arredondadas */
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="_txt_treino">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Treino de</string>
    </property>
   </widget>
   <widget class="QLabel" name="_txt_terca_user">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>481</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="_txt_sexta_user">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>481</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="_te_comentario">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>551</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="_txt_comentario">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Observações:</string>
    </property>
   </widget>
   <widget class="QLabel" name="_txt_quarta">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Quarta</string>
    </property>
   </widget>
   <widget class="QPushButton" name="_btn_salvar">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>520</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Salvar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="_btn_quinta">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>260</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="recursos.qrc">
      <normaloff>:/img/Imagens/imagem_plus.png</normaloff>:/img/Imagens/imagem_plus.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="_txt_segunda">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Segunda</string>
    </property>
   </widget>
   <widget class="QPushButton" name="_btn_terca">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>140</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="recursos.qrc">
      <normaloff>:/img/Imagens/imagem_plus.png</normaloff>:/img/Imagens/imagem_plus.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="_txt_quarta_user">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>481</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="_txt_quinta_user">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>481</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="_txt_quinta">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Quinta</string>
    </property>
   </widget>
   <widget class="QPushButton" name="_btn_segunda">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>80</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="recursos.qrc">
      <normaloff>:/img/Imagens/imagem_plus.png</normaloff>:/img/Imagens/imagem_plus.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="_btn_quarta">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>200</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="recursos.qrc">
      <normaloff>:/img/Imagens/imagem_plus.png</normaloff>:/img/Imagens/imagem_plus.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="_txt_terca">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Terça</string>
    </property>
   </widget>
   <widget class="QPushButton" name="_btn_sexta">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>320</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="recursos.qrc">
      <normaloff>:/img/Imagens/imagem_plus.png</normaloff>:/img/Imagens/imagem_plus.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="_txt_sexta">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Sexta</string>
    </property>
   </widget>
   <widget class="QLabel" name="_txt_segunda_user">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>481</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="_txt_usuario">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="_txt_imagem">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="recursos.qrc">:/img/Imagens/imagem_gym.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="_btn_sexta_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>320</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="recursos.qrc">
      <normaloff>:/img/Imagens/imagem_less.png</normaloff>:/img/Imagens/imagem_less.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="_btn_segunda_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>80</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="recursos.qrc">
      <normaloff>:/img/Imagens/imagem_less.png</normaloff>:/img/Imagens/imagem_less.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="_btn_terca_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>140</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="recursos.qrc">
      <normaloff>:/img/Imagens/imagem_less.png</normaloff>:/img/Imagens/imagem_less.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="_btn_quarta_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>200</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="recursos.qrc">
      <normaloff>:/img/Imagens/imagem_less.png</normaloff>:/img/Imagens/imagem_less.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="_btn_quinta_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>260</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="recursos.qrc">
      <normaloff>:/img/Imagens/imagem_less.png</normaloff>:/img/Imagens/imagem_less.png</iconset>
    </property>
   </widget>
   <zorder>_txt_sexta_user</zorder>
   <zorder>_txt_quarta_user</zorder>
   <zorder>_txt_segunda_user</zorder>
   <zorder>_txt_quinta_user</zorder>
   <zorder>_txt_terca_user</zorder>
   <zorder>_txt_treino</zorder>
   <zorder>_te_comentario</zorder>
   <zorder>_txt_comentario</zorder>
   <zorder>_btn_salvar</zorder>
   <zorder>_btn_quinta</zorder>
   <zorder>_btn_terca</zorder>
   <zorder>_btn_segunda</zorder>
   <zorder>_btn_quarta</zorder>
   <zorder>_btn_sexta</zorder>
   <zorder>_txt_usuario</zorder>
   <zorder>_txt_segunda</zorder>
   <zorder>_txt_imagem</zorder>
   <zorder>_txt_sexta</zorder>
   <zorder>_txt_quinta</zorder>
   <zorder>_txt_quarta</zorder>
   <zorder>_txt_terca</zorder>
   <zorder>_btn_sexta_2</zorder>
   <zorder>_btn_segunda_2</zorder>
   <zorder>_btn_terca_2</zorder>
   <zorder>_btn_quarta_2</zorder>
   <zorder>_btn_quinta_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>597</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
